rosdep/base.yaml

```diff
@@ -2649,6 +2649,15 @@ libcairo2-dev:
   opensuse: [cairo-devel]
   rhel: [cairo-devel]
   ubuntu: [libcairo2-dev]
+libcamera:
+  debian:
+    sid: [libcamera-dev]
+  fedora:
+    "36": [libcamera]
+  gentoo: ['media-libs/libcamera']
```

Comment:
Can you point to this package, [I can't find it](https://packages.gentoo.org/packages/search?q=libcamera).

Reply:
I found it here: http://gpo.zugaina.org/media-libs/libcamera.